docs/modules/usage/how-to/github-action.md

```diff
@@ -0,0 +1,15 @@
+# Using the OpenHands GitHub Action
+
+This guide explains how to use the OpenHands GitHub Action, both within the OpenHands repository and in your own projects.
+
+## Using the Action in the OpenHands Repository
+
+To use the OpenHands GitHub Action in the OpenHands repository:
+
+1. Create an issue in the repository.
+2. Add the `fix-me` label to the issue.
+3. The action will automatically trigger and attempt to resolve the issue.
```

Comment:
This one depends on certain permissions, I assume. I think we could state which specifically.

I see it works for me, but non-collaborators cannot label issues, if I recall correctly?